packages/runtime/src/helpers/edge.ts

```diff
@@ -451,6 +463,7 @@ export const writeEdgeFunctions = async ({
           name: edgeFunctionDefinition.name,
           pattern: dataRoute,
           cache: usesAppDir ? 'manual' : undefined,
+          generator: `${nextjsPluginVersion}`,
```

Comment:
This doesn't need to be a template string now and you can just use the variable.

```suggestion
      generator: nextjsPluginVersion`,
```